Cargo.toml

```diff
@@ -27,6 +27,7 @@ prost-types = "0.13.3"
 #libp2p = { path = "../../rust-libp2p/libp2p" }
 #libp2p-core = { path = "../../rust-libp2p/core" }
 #libp2p-swarm = { path = "../../rust-libp2p/swarm" }
+system-configuration = { git = "https://github.com/mullvad/system-configuration-rs", tag = "v0.6.1" }
```

Comment:
is this used? we generally don't allow git based deps as they are no-go when publishing to crates